packages/core/src/codewhispererChat/tools/fsRead.ts

```diff
@@ -122,4 +136,17 @@ export class FsRead {
             },
         }
     }
+
+    private sanitizePath(inputPath: string): string {
+        let sanitized = inputPath.trim()
+
+        if (sanitized.startsWith('~')) {
+            sanitized = path.join(fs.getUserHomeDir(), sanitized.slice(1))
+        }
+
+        if (!path.isAbsolute(sanitized)) {
+            sanitized = path.resolve(sanitized)
+        }
+        return sanitized
+    }
```

Comment:
Can we pull this out to `toolShared`? It's needed for FsWrite as well

Reply:
Done